site/public/js/server.js

```diff
@@ -285,6 +290,41 @@ function newUploadCourseMaterialsForm() {
     $('#overwrite-materials-flag').remove();
 }
 
+function newUploadBanner() {
+
+    createArray(1);
+
+    var files = [];
+
+    $('.popup-form').css('display', 'none');
+    var form = $("#upload-banner");
+
+    $('[name="existing-file-list"]', form).html('');
+    $('[name="existing-file-list"]', form).append('<b>'+JSON.stringify(files)+'</b>');
```

Comment:
Please address the other comments that were here. This could potentially cause security issues and it better to put in text content rather than raw html.